### ISSUE TITLE
Error code 9021 is undocumented

### ISSUE THREAD

**Issue by MortenSoerensen** [OfficeDev/office-js-docs-pr#1555]
## Article URL
In the article https://docs.microsoft.com/en-us/office/dev/add-ins/reference/javascript-api-for-office-error-codes the is no description of the error with code 9021.

## Issue

I have experienced the error in Outlook on Windows in cached mode.
The error came in message compose mode, when saving a mail item as a draft.

The error is also mentioned in https://github.com/OfficeDev/office-js/issues/373 where the error was returned when saving a calendar item.
The error message is: "Connection error occurred while trying to save the item on the server.",
The error name is: "SaveError"

**Comment by ElizabethSamuel-MSFT** [comment]
@MortenSoerensen Thanks for raising this issue. We'll document this error.

@exextoc This may be a bug since in cached mode, it seems that the message should save to the cache and not be trying to save to the server. Can you investigate?

I'll resolve the documentation issue then transfer to office-js if need be.

Thanks.

**Comment by exextoc** [comment]
Thanks for reporting this. We looked at the place where the error is returned in Outlook, but it doesn't represent a singular error condition. If the user is also unable to save the item, Outlook may display a more specific reason to the user for why the item can't be saved.

If you are able to reproduce this yourself, we can try the same steps to reproduce the issue and investigate why the error is returned in that specific case.

**Comment by MortenSoerensen** [comment]
Hi @exextoc  and @ElizabethSamuel-MSFT 
One of the scenarios i get the error is when I try to call saveAsync on a draft that has already been sent using the REST API.
The reason i want to do that, is because I made changes to the mail item on the server using the REST API before sending it, and I want to prevent Outlook from displaying an unsaved changes to the draft alert when I close the mail window.
I think I get the error because the mail item has already been moved to the sent folder and in that case the error should just be ignored.
I will close the issue, because I see you have updated the https://docs.microsoft.com/en-us/office/dev/add-ins/reference/javascript-api-for-office-error-codes page with the error code and description now.
Regards Morten
